content/blog/cdcon+gitopscon-at-open-source-summit/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 📢 GitOpsCon relocates to Open Source Summit
+date: "2022-12-16"
+description: "GitOpsCon relocates to Open Source Summit"
+author: Scott Rigby
+thumbnail: images/gitopscon-opengitops-blog-banner.png
+---
+
+On December 14th, the CNCF [announced changes to how co-located events will work](https://www.cncf.io/blog/2022/12/14/important-updates-for-2023-kubecon--cloudnativecon-and-co-located-events/) moving forward, starting with KubeCon+CloudNativeCon 2023 EU.
+
+With this announcement, we are happy to announce that GitOpsCon is joining forces with CdCon and moving from KubeCon + CloudNativeCon to Open Source Summit starting in 2023!
```

Comment:
CdCon -> cdCon based on https://events.linuxfoundation.org/cdcon/

Reply:
@sympatheticmoose Fixed in #109 with co-author credit to you 🙌